apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -258,8 +301,8 @@ export default connect(
             }}
           >
             {experiments.isEnabled('react-inspector')
-              ? this.displayOutputToConsole()
-              : this.props.logOutput}
```

Comment:
Since logOutput holds all the input, output, and any error messages if there is any, I assumed we would want it passed in props? Could you explain more about why logOutput should be removed from props?